### DISCUSSION TITLE
Login using either username/email

### DISCUSSION THREAD

**Discussion by iJustErik** [NamelessMC/Nameless#2126] [Ideas]
Could we have the option to use email AND password for logging in? Maybe it says username/email, but you can login using both

**Comment by nadecancode** [comment]
Agree, just check if the string contains @ to specify its email or username

**Comment by partydragen** [comment]
This feature has been added in https://github.com/NamelessMC/Nameless/commit/36ea39ff3d67a453de2ff601a4ef84c42d9289ea
